google/datalab/contrib/pipeline/_pipeline.py

```diff
@@ -104,6 +107,12 @@ def py(self):
     return Pipeline._imports + default_args + dag_definition + \
         task_definitions + up_steam_statements
 
+  @staticmethod
+  def _get_datetime_expr(date_string):
```

Comment:
Also, why don't we evaluate the expression instead of returning a string to be evaluated later?